test/box-luatest/gh_6539_log_user_space_empty_or_nil_select_test.lua

```diff
@@ -7,24 +7,39 @@ local g = t.group()
 g.before_all(function()
     g.server = server:new{alias = 'dflt'}
     g.server:start()
+    g.server:eval('require("log").internal.ratelimit.cfg.enable = false')
```

Comment:
Nit: please use `g.server.exec` instead of `eval`.

Reply:
What's it worth? I prefer using `eval` for one-liners.

Reply:
Agreed on the following: use `eval` only when API calls are needed to be generated.